### ISSUE TITLE
Attempt to Cache Qemu Stage of Build

### ISSUE THREAD

**Issue by ghickman** [py3minepi/py3minepi-legacy#41]
Following on from discussion [here](https://github.com/py3minepi/py3minepi/pull/28#issuecomment-57637110) we should look into a way to cache the Qemu image so it's not rebuilt on every test run!

**Comment by doismellburning** [comment]
https://github.com/py3minepi/arm-chroot

```
18:01 < doismellburning> I am so very confused about environment variables
18:02 < doismellburning> I have set some AWS keys in https://travis-ci.org/py3minepi/arm-chroot/settings/env_vars
18:02 < doismellburning> but from readinb http://docs.travis-ci.com/user/uploading-artifacts/ and http://docs.travis-ci.com/user/encryption-keys/ and http://docs.travis-ci.com/user/environment-variables/ I am very unclear how to use them 
                         to upload my tarballs to S3
```

**Comment by doismellburning** [comment]
Attempting to be fixed in #53
